openmdao/docs/openmdao_book/features/building_blocks/components/ks_comp.ipynb

```diff
@@ -394,6 +394,150 @@
    "source": [
     "assert_near_equal(prob.get_val('ks.KS', indices=0), np.amax(prob.get_val('x')), tolerance=1e-8)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Computing Min and Max of KS**\n",
```

Comment:
suggested text:

**Example: KSComp for aggregating a constraint with a lower bound. 

When you are using the KSComp to aggregate a constraint with a lower bound, set the "lower_flag"
to True, and it will create a constraint that is satisfied when it is greater than the value
specified in "upper". "Because KSComp is a smooth max function, you can pass in any values to 
\"g\" and it will approximate both the min and max.